app/forms/topic_archival_form.rb

```diff
@@ -2,16 +2,12 @@ class TopicArchivalForm
   include ActiveModel::Model
   attr_accessor :tag, :successor, :successor_path
```

Comment:
```suggestion
  attr_accessor :tag, :successor_path
```

Shall we also remove anything related to the `successor` param incl. from [the controller](https://github.com/alphagov/collections-publisher/blob/af60dd4/app/controllers/topics_controller.rb#L78)

Reply:
Good spot thank you